terraform/deployments/tfc-configuration/variables-integration.tf

```diff
@@ -55,6 +55,22 @@ module "variable-set-integration" {
   }
 }
 
+module "variable-set-chat-integration" {
+  source = "./variable-set"
+
+  name = "chat-integration"
+
+  tfvars = {
+    chat_redis_cluster_apply_immediately          = true
+    chat_redis_cluster_automatic_failover_enabled = false
+    chat_redis_cluster_engine_version             = "6.x"
```

Comment:
Shall we use version 7? That's what we use in dev and on Heroku.

Reply:
Changed engine version to 7.1